build/constants.yaml

```diff
@@ -110,8 +110,8 @@
     final-stretch-dot-net-70-sdk-version: 7.0.100
     final-stretch-dot-net-core-app-70-run-time-version: 7.0.0
     final-stretch-aspnet-core-app-70-run-time-version: 7.0.0
-    final-stretch-node14-version: 14.19.1
-    final-stretch-node16-version: 16.14.2
+    final-stretch-node14-version: 14.21.3
+    final-stretch-node16-version: 16.20.0
```

Comment:
Why do we need to update these stretch versions?

Reply:
actually, I updated the versions and ran all the pipelines before remembering that we don't need to update the stretch versions. However, as Paul suggested "app service still has not integrated our changes to dynamically install SDKs from the storage accounts corresponding to the debian version", so thought to keep the updated stretch version (as we are running fast towards the deadline). 
If you suggest to revert the changes, I will do asap.

Reply:
As far as I know app service is still using old images from August. Those old images are pulling the stretch versions of languages, so we still need to upload these versions. Since we are uploading new stretch versions, the `final-stretch` version also needs to get updated since it is used in tests and such where we use a stretch image and dynamically install something like node 16 (without specifying the exact version), and expect it to install the final stretch version: https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs#L167